projects/plugins/jetpack/modules/custom-post-types/portfolios.php

```diff
@@ -690,127 +828,135 @@ static function portfolio_query( $atts ) {
 	 * The Portfolio shortcode loop.
 	 *
 	 * @todo add theme color styles
+	 *
+	 * @param array $atts Shortcode attributes.
+	 *
 	 * @return html
 	 */
-	static function portfolio_shortcode_html( $atts ) {
-
-		$query = self::portfolio_query( $atts );
+	private static function portfolio_shortcode_html( $atts ) {
+		$query                  = self::portfolio_query( $atts );
 		$portfolio_index_number = 0;
 
 		ob_start();
 
-		// If we have posts, create the html
-		// with hportfolio markup
+		/*
+		 * If we have posts, create the html
+		 * with hportfolio markup
+		 */
 		if ( $query->have_posts() ) {
-
-			// Render styles
-			//self::themecolor_styles();
-
-		?>
+			/*
+			 * Render styles
+			 * See self::themecolor_styles();
+			 */
+			?>
 			<div class="jetpack-portfolio-shortcode column-<?php echo esc_attr( $atts['columns'] ); ?>">
-			<?php  // open .jetpack-portfolio
+			<?php
+			// open .jetpack-portfolio.
 
 			// Construct the loop...
 			while ( $query->have_posts() ) {
 				$query->the_post();
 				$post_id = get_the_ID();
 				?>
-				<div class="portfolio-entry <?php echo esc_attr( self::get_project_class( $portfolio_index_number, $atts['columns'] ) ); ?>">
+				<div class="portfolio-entry <?php echo esc_attr( self::get_project_class( $portfolio_index_number, (int) $atts['columns'] ) ); ?>">
```

Comment:
This has to be an int for later, but shortcode attributes are typically strings.